apple_sign_in.gemspec

```diff
@@ -1,34 +1,41 @@
 # frozen_string_literal: true
 
-require_relative 'lib/apple_sign_in/version'
+require_relative 'lib/apple_sign_in/base/version'
 
 Gem::Specification.new do |spec|
   spec.name          = 'apple_sign_in'
-  spec.version       = AppleSignIn::VERSION
-  spec.authors       = ['fedeagripa']
-  spec.email         = ['federicogagripa@gmail.com']
+  spec.version       = AppleSignIn::Base::VERSION
+  spec.authors       = ['Timothy Peraza']
+  spec.email         = ['timothy@rootstrap.com']
 
-  spec.summary       = ''
-  spec.description   = ''
-  spec.homepage      = ''
+  spec.summary       = 'A gem that implement an integration with apple sign in'
```

Comment:
How about describing the gem with `Integration with Sign In with Apple authentication`?